hardware_interface/include/hardware_interface/actuator_interface.hpp

```diff
@@ -426,13 +538,20 @@ class ActuatorInterface : public rclcpp_lifecycle::node_interfaces::LifecycleNod
   std::vector<CommandInterface::SharedPtr> unlisted_commands_;
 
   rclcpp_lifecycle::State lifecycle_state_;
+  std::unique_ptr<realtime_tools::AsyncFunctionHandler<return_type>> async_handler_;
 
 private:
   rclcpp::node_interfaces::NodeClockInterface::SharedPtr clock_interface_;
   rclcpp::Logger actuator_logger_;
   // interface names to Handle accessed through getters/setters
   std::unordered_map<std::string, StateInterface::SharedPtr> actuator_states_;
   std::unordered_map<std::string, CommandInterface::SharedPtr> actuator_commands_;
+  enum class TriggerType
+  {
+    READ,
+    WRITE
+  };
```

Comment:
I think it would be a good first issue to not repeat this enum in all classes right?

Reply:
Yes! Totally. It would be great to reuse this info.